template.compose.yaml

```diff
@@ -57,11 +57,25 @@ services:
       - MINIO_ACCESS_KEY=minioadmin
       - MINIO_SECRET_KEY=minioadmin
       - FRONT_URL=http://front:8080
-      - INIT_WORKSPACE=demo-tracker
       - MODEL_ENABLED=*
       - ACCOUNTS_URL=http://${SERVER_ADDRESS}:3000
       - ACCOUNT_PORT=3000
     restart: unless-stopped
+  workspace:
+    image: hardcoreeng/workspace:${HULY_VERSION}
+    links:
+      - mongodb
+      - minio
+    environment:
```

Comment:
Please also add NOTIFY_INBOX_ONLY=true

Reply:
done

Reply:
Cannot see the change here. Did you forget to commit/push? :)